docs/overview.md

```diff
@@ -0,0 +1,6 @@
+# Overview 🎯
+
+Wattles is a strongly typed Dart ORM that does not require generating code to function. It is inspired by many tools including [TypeORM](https://typeorm.io/), [Mikro ORM](https://mikro-orm.io/), and [Prisma](https://www.prisma.io/).
+
+The goal of Wattles is to help developers talk to their databases while using a strongly typed API. Wattles main focus is to be a true ORM, create a virtual object database that maps the data to schemas while keeping relations intact. It does not, however, provide a way to create custom queries specifically tailored to your use case or create migrations for your database.
```

Comment:
Will custom queries not be supported at all? At least a simple way to make then should be included, even if the query is just a string and the return is just a list of map.

Reply:
That would heavily depend on the driver behind it, while discussing it with @pstromberg98 we found out that the query builder can still be quite powerful, using custom types, but for now I will scope the requirements to only a true ORM.